package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/lambda-tools",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
This project is setup with Travis to publish on tagged commits on master. This means that we need to bump versions separately after merges. This also has the added benefit of making it easier to coordinate multiple parallel changes.

Reply:
Good to know. I removed. it.